pkg/gce-pd-csi-driver/controller.go

```diff
@@ -398,6 +411,10 @@ func (gceCS *GCEControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 		}
 		disk, err = createRegionalDisk(ctx, gceCS.CloudProvider, name, zones, params, capacityRange, capBytes, snapshotID, volumeContentSourceVolumeID, multiWriter)
 		if err != nil {
+			// Emit metric for error
+			if strings.Contains(params.DiskType, "pd") {
+				gceCS.Metrics.RecordOperationErrorMetrics(pdcsiDriverName, "/csi.v1.Controller/CreateVolume", err, existingDisk.GetPDType())
+			}
```

Comment:
After creating the disk we should also care about the next isDiskReady() for Hyperdisk.